addons/test_mail_full/models/test_mail_models_mail.py

```diff
@@ -56,7 +56,6 @@ class MailTestRating(models.Model):
     company_id = fields.Many2one('res.company', 'Company')
     customer_id = fields.Many2one('res.partner', 'Customer')
     email_from = fields.Char('From', compute='_compute_email_from', precompute=True, readonly=False, store=True)
-    mobile_nbr = fields.Char('Mobile', compute='_compute_mobile_nbr', precompute=True, readonly=False, store=True)
```

Comment:
Why changing this ? It is a test model, not related to partner, just make it a classic field for test purpose

Reply:
Although its test model, but [Here](https://github.com/odoo/odoo/blob/7f0017c4728bac9c721e2dd13afb01bc06d217ea/addons/test_mail_full/models/test_mail_models_mail.py#L71-L78) and [Here](https://github.com/odoo/odoo/blob/7f0017c4728bac9c721e2dd13afb01bc06d217ea/addons/test_mail_sms/models/test_mail_sms_models.py#L44-L47) we get that mobile from res.partner, if not then `False`.. so better to remove it..?

Reply:
would say keep the field, set it as a manual field, and check in tests we effectively search for value on both fields (will probably need some manual update of some records), as the purpose is also to check the support of multi fields (like old partner model).

Reply:
note that maybe for mail test rating it is not necessary ... did that impact test ? maybe keeping two fields on mail test sms is sufficient ?

Reply:
well I checked and did not found anything where we use mobile_nbr in `mail.test.rating` (test_mail_full), I might be wrong tho .. So yes keeping that in `test_mail_sms` is sufficient :smiley: